### PR TITLE
Add doc and tools for using Theia, and doc formatting

### PR DESCRIPTION
**Pull request by oleg-alexandrov**
Changes:

 - Added a wrapper around the Theia tool for building a map. 
 - Fixed some issues in the localization docs to make links to other subpages work well with GitHub's markup. 
 - Updated the bumble calibration for sci cam, haz cam, and extrinsics among these and to nav cam. Did not change the nav cam calibration.

### CODE REVIEW

File: localization/sparse_mapping/readme.md [hunk 1]
```diff
@@ -50,46 +51,48 @@ Usually the bags are acquired at a very high frame rate, and they are
...
+robot's IP address.
 
 ### Merging bags
 
 The bags created on the ISS are likely split into many smaller bags,
 for easy and reliability of transfer. Those can be merged into one bag
 as follows:
 
-    astrobee_build/devel/lib/localization_node/merge_bags \
+    astrobee/build/devel/lib/localization_node/merge_bags \
```
Comment (marinagmoreira) [inline localization/sparse_mapping/readme.md:76]: should be astrobee/devel/lib/localization_node/merge_bags according to the default setup

File: localization/sparse_mapping/readme.md [hunk 2]
```diff
@@ -50,46 +51,48 @@ Usually the bags are acquired at a very high frame rate, and they are
...
       -output_bag <output bag> <input bags>
 
 ### Extracting images
 
 To extract images from a bag file:
 
-     extract_image_bag <bagfile.bag> -use_timestamp_as_image_name \
-       -image_topic /hw/cam_nav -output_directory <output dir>
+    astrobee/build/devel/lib/localization_node/extract_image_bag   \
```
Comment (marinagmoreira) [inline localization/sparse_mapping/readme.md:83]: same here

**Review by marinagmoreira — CHANGES_REQUESTED** [review]

**Comment by oleg-alexandrov** [comment]
I implemented Marina's suggestions (also in many other places) so that the correct paths to the "devel" directory is used. 

Sorry this pull request became too big.

**Review by marinagmoreira — APPROVED** [review]

**Comment by oleg-alexandrov** [comment]
I pulled this in myself. I have another pull request to make, and this way staying in the way and was approved.

**Comment by bcoltin** [comment]
Once it's been approved by a reviewer it's fine to pull it in yourself. Thanks!

**Pull request merged**